src/controllers/user.controller.js

```diff
@@ -0,0 +1,84 @@
+const userService = require('../services/user.service.js');
+
+const get = (_req, res) => {
+  res.status(200).send(userService.getAll());
+};
+
+const getById = (req, res) => {
+  const { id } = req.params;
+
+  const user = userService.getById(+id);
+
+  if (!user) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.send(user);
+};
+
+const create = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    return res.sendStatus(400);
+  }
+
+  const newUser = userService.create(name);
+
+  // res.statusCode = 201;
+
+  // res.send(newUser);
+  res.status(201).send(newUser);
+};
+
+const remove = (req, res) => {
+  const { id } = req.params;
+
+  if (!id) {
+    res.sendStatus(200);
```

Comment:
Returning a 200 status code when `id` is not provided might not be appropriate. Consider using a 400 status code to indicate a bad request.